dev/serve.vue

```diff
@@ -29,7 +29,7 @@ export default defineComponent({
       donation_url="https://www.classy.org/give/313412/#!/donation/checkout?c_src=website&c_src2=top-of-page-banner"
     />
     <cc-global-header
-      base_url="http://localhost:8000/wp-json/ccnavigation-header/menu"
+      base_url="http://127.0.0.1:8000"
```

Comment:
Let's follow the convention of using kebab-case for component attributes/properties, which will automatically convert the property to camelCase in our code:

https://v3.vuejs.org/guide/component-props.html#prop-casing-camelcase-vs-kebab-case